test/jdk/lib/client/ExtendedRobot.java

```diff
@@ -57,11 +57,6 @@ public class ExtendedRobot extends Robot {
 
     private final int syncDelay = DEFAULT_SYNC_DELAY;
 
-    //TODO: uncomment three lines below after moving functionality to java.awt.Robot
-    //{
-    //    syncDelay = AccessController.doPrivileged(new GetIntegerAction("java.awt.robotdelay", DEFAULT_SYNC_DELAY));
-    //}
-
```

Comment:
Not related to this PR changes but since ExtendedRobot was mentioned here, I went through Robot.java and it has few SecurityException in javadoc [(as here)](https://github.com/openjdk/jdk/blob/db85090553ab14a84c3ed0a2604dd56c5b6e6982/src/java.desktop/share/classes/java/awt/Robot.java#L506C1-L507C43). 
Will changes to Robot.java be handled as separate PR + CSR ?

Reply:
> Not related to this PR changes but since ExtendedRobot was mentioned here, I went through Robot.java and it has few SecurityException in javadoc [(as here)](https://github.com/openjdk/jdk/blob/db85090553ab14a84c3ed0a2604dd56c5b6e6982/src/java.desktop/share/classes/java/awt/Robot.java#L506C1-L507C43). Will changes to Robot.java be handled as separate PR + CSR ?

We still use SecurityException in java.awt.Robot too. It is still needed for when the *desktop* denies access.

Reply:
Makes sense. Thanks for clarifying.